scrapy/downloadermiddlewares/redirect.py

```diff
@@ -34,6 +34,8 @@ def _redirect(self, redirected, request, spider, reason):
             redirected.meta['redirect_ttl'] = ttl - 1
             redirected.meta['redirect_urls'] = request.meta.get('redirect_urls', []) + \
                 [request.url]
+            redirected.meta['redirect_codes'] = request.meta.get('redirect_codes', []) + \
```

Comment:
in case of MetaRefreshMiddleware instead of an http code a string can be passed as a reason. What do you think about renaming this meta key to e.g. redirect_reasons?

Reply:
Yeah nice idea @kmike :)